addon/modifiers/scroll-into-view.js

```diff
@@ -45,18 +45,22 @@ function scrollIntoView(element, positional, named = {}) {
 
   shouldScrollPromise.then((shouldScrollValue) => {
     if (shouldScrollValue && element && !hasBeenRemoved) {
+      let { behavior = 'auto' } = options || {};
+      behavior =
+        behavior === 'smooth' &&
```

Comment:
What should happen here if `behavior` wasn't set at all? Does the `auto` default automatically handle `prefers-reduced-motion` in browsers?

Reply:
So there is no change here compared to the original implementation. Wherein, for non-offset condition, we do not set the behavior to anything if `options` do not have a `behavior` in it. But for offset condition, we do set it to default `'auto'`.

If behavior is 'auto' I did not touch it, because then its mostly CSS dependent and the user of the modifier is then responsible of how top manage the behavior and not the modifier itself.

In my explorations for this, I discovered that Firefox handles this automatically but Chrome does not. This condition that I have added is essentially in line with A11y recommendations.
The idea behind this implementation is essentially to keep it in a way that we can clean this out once (if) chrome starts to support it.